src/shadowbox/README.md

```diff
@@ -1,203 +1,169 @@
+     >
+     > - **Debug Image:**
+     >
+     >   ```sh
+     >   docker run --rm -it --entrypoint=sh localhost/outline/shadowbox
+     >   ```
+     >
+     > - **Debug Running Container:**
+     >
+     >   ```sh
+     >   docker exec -it shadowbox sh
+     >   ```
+     >
+     > - **Cleanup Dangling Images:**
+     >
+     >   ```sh
+     >   docker rmi $(docker images -f dangling=true -q)
+     >   ```
 
-```
-npm run action shadowbox/docker/start
-```
+1. **Send a Test Request**
 
-You should be able to successfully query the management API:
-
-```
-curl --insecure https://[::]:8081/TestApiPrefix/server
-```
-
-To build the image only:
-
-```
-npm run action shadowbox/docker/build
-```
-
-Debug image:
-
-```
-docker run --rm -it --entrypoint=sh localhost/outline/shadowbox
-```
-
-Or a running container:
-
-```
-docker exec -it shadowbox sh
-```
-
-Delete dangling images:
-
-```
-docker rmi $(docker images -f dangling=true -q)
-```
+   ```sh
+   curl --insecure https://[::]:8081/TestApiPrefix/server
+   ```
 
 ## Access Keys Management API
 
-In order to utilize the Management API, you'll need to know the apiUrl for your Outline server.
-You can obtain this information from the "Settings" tab of the server page in the Outline Manager.
-Alternatively, you can check the 'access.txt' file under the '/opt/outline' directory of an Outline server. An example apiUrl is: https://1.2.3.4:1234/3pQ4jf6qSr5WVeMO0XOo4z.
-
-See [Full API Documentation](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/Jigsaw-Code/outline-server/master/src/shadowbox/server/api.yml).
-The OpenAPI specification can be found at [api.yml](./server/api.yml).
+1. **Find your `apiUrl`:** (e.g. `https://1.2.3.4:1234/3pQ4jf6qSr5WVeMO0XOo4z`)
 
-### Examples
+   - **Deployed with the Installation Script:** Run `grep "apiUrl" /opt/outline/access.txt | cut -d: -f 2-`
 
-Start by storing the apiURL you see see in that file, as a variable. For example:
+   - **Deployed with the Outline Manager:** Check the "Settings" tab.
 
-```
-API_URL=https://1.2.3.4:1234/3pQ4jf6qSr5WVeMO0XOo4z
-```
+   - **Local Deployments from Source:** `https://[::]:8081/TestApiPrefix`
 
-You can then perform the following operations on the server, remotely.
+1. **API Examples:**
 
-List access keys
+   Replace `$API_URL` with your actual `apiUrl`.
 
-```
-curl --insecure $API_URL/access-keys/
-```
+   - **List access keys:** `curl --insecure $API_URL/access-keys/`
 
-Create an access key
+   - **Create an access key:** `curl --insecure -X POST $API_URL/access-keys`
 
-```
-curl --insecure -X POST $API_URL/access-keys
-```
+   - **Get an access key (e.g. ID 1):** `curl --insecure $API_URL/access-keys/1`
 
-Get an access key (e.g. get access key 1)
+   - **Rename an access key:** `curl --insecure -X PUT -F 'name=albion' $API_URL/access-keys/2/name`
 
-```
-curl --insecure $API_URL/access-keys/1
-```
+   - **And more...**
```

Comment:
The idea was to have the primary methods used by providers listed on this page.

Reply:
Put the extra ones back in.